secure-api-gateway-ob-uk-rcs-consent-store/secure-api-gateway-ob-uk-rcs-consent-store-api/src/test/java/com/forgerock/sapi/gateway/rcs/consent/store/api/account/AccountAccessConsentValidationHelpers.java

```diff
@@ -36,6 +35,20 @@ public class AccountAccessConsentValidationHelpers {
     /**
      * Validates that a created {@link AccountAccessConsent} against the original {@link CreateAccountAccessConsentRequest}
      */
+    public static void validateCreateConsentAgainstCreateRequest(AccountAccessConsent consent,
+                                                                 CreateAccountAccessConsentRequest createAccountAccessConsentRequest, OBVersion obVersion) {
+        assertThat(consent.getId()).isNotEmpty();
+        assertThat(consent.getStatus()).isEqualTo(OBExternalRequestStatus1Code.AWAITINGAUTHORISATION.toString());
+        assertThat(consent.getApiClientId()).isEqualTo(createAccountAccessConsentRequest.getApiClientId());
+        assertThat(consent.getRequestObj()).isEqualTo(createAccountAccessConsentRequest.getConsentRequest());
+        assertThat(consent.getRequestVersion()).isEqualTo(obVersion);
+        assertThat(consent.getResourceOwnerId()).isNull();
+        assertThat(consent.getAuthorisedAccountIds()).isNull();
+
+        assertThat(consent.getCreationDateTime()).isBefore(new Date());
+        assertThat(consent.getStatusUpdateDateTime()).isEqualTo(consent.getCreationDateTime());
+    }
+
     public static void validateCreateConsentAgainstCreateRequest(AccountAccessConsent consent,
                                                                  CreateAccountAccessConsentRequest createAccountAccessConsentRequest) {
```

Comment:
Looks like the change above is just adding a version param to this method and it looks like the original method is no longer used so it should be removed.